src/Helpers/Composer/PackageHelper.php

```diff
@@ -70,5 +71,41 @@ private function getComposer(InputInterface $input, OutputInterface $output) {
         return $composer;
     }
 
+    /**
+     * @param PackageHelper $packageHelper
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return bool
+     */
+    public static function isPhpUnitInstalled(
```

Comment:
i missed the fact that i was adding statics to a helper. these started out as inline calls to the package helper, just like your other classes. i saw the duplicate magic strings and thought maybe they should be put in a reusable place. so i made them static on my command-class, but of course it wouldn't make sense for other commands to call TravisCi for the PhpUnit package helper. thus, a wrong decision is born.

even as instance-methods i certainly don't want this helper to grow for forever.

stateless-statics are, of course, no replacement for an abstraction, and i don't want to provide an abstraction too early, before we really know what it should be or what it needs. i'll slurp these back into private methods on my command and we'll let the pattern grow organically.

if the decorator you are speaking of must live in the TravicCi command namespace to decrease chances of coupling dependency versions to commands, then i don't see the point.